README.md

```diff
@@ -25,47 +27,40 @@ First of all, clone the github repository: `git clone https://github.com/project
 Simply run
 
 ```
-pip install .
-```
-
-If you want to do some changes to the code, I suggest to do:
-
-```
-cd atomicapp
-export PYTHONPATH=`pwd`:$PYTHONPATH
-alias atomicapp="python `pwd`/atomicapp/cli/main.py"
+make install
 ```
 
-### Build
+### Build an Atomic App
 ```
 docker build -t [TAG] .
 ```
 
-Just a call to Docker to package up the application and tag the resulting image.
+Use 'docker build' to package up the application and tags the resulting image.
 
 ### Install and Run
 ```
-atomicapp [--dry-run] [-a answers.conf] install|run [--recursive] [--update] [--destination DST_PATH] APP|PATH
+atomicapp [--dry-run] [-a answers.conf] fetch|run|stop [--recursive] [--update] [--provider docker] [--destination DST_PATH] APP|PATH
```

Comment:
There is also genanswers. I think maybe we should not put all of the options here because it gets too long.